conf/code-lists/pointer-mapping.csv

```diff
@@ -102,4 +102,6 @@
 42A.D014,suppressed
 42A.67A.68A.02A.D031,suppressed
 42A.67A.68A.02A.D006,suppressed
-42A.03A.$1.226,suppressed
\ No newline at end of file
+42A.03A.$1.226,suppressed
+48A.67A.68A.02A.D031,suppressed
+48A.67A.68A.02A.D006,suppressed
```

Comment:
Why are we suppressing these pointers. They are pointing to Declaration/GoodsShipment/GovernmentAgencyGoodsItem/AdditionalDocument/CategoryCode, why are we not directing users there?